doc/nrf/ug_thread_commissioning.rst

```diff
@@ -52,10 +52,35 @@ For details about scenarios that include devices with multiple roles, see `Threa
 
 See :ref:`thread_ot_commissioning_cli` for the list of commands used to assign the commissioning roles.
 
+.. _thread_ot_device_types:
+
+Device types
+============
+
+OpenThread distinguishes between different device types depending on what roles a device can take in the Thread network.
+
+Full Thread Device (FTD)
+  An FTD can be both a router and an end device.
+  Its radio is always on.
+
+Minimal Thread Device (MTD)
+  An MTD is always an end device.
+  It forwards all messages to its parent.
+
+  There are two important subtypes:
+
+  Minimal End Device (MED)
+    An MED keeps its transceiver always on.
```

Comment:
not sure about about "an" here and below. We usually pronounce it just med/sed (not em ee dee/es ee dee). But I/m not that good in English. Same goes for fed/reed. But it is ok to have "an" for FTD/MTD as we pronounce it "ef tee dee"/"em tee dee".

Reply:
Thread Specification confirms your point:

> A SED has its radio turned off during idle periods and it wakes periodically to communicate with its parent.

> FTDs may be a REED or a FED, MTDs may be a MED or a SED.

Reply:
Thanks, will update. :)